### PR TITLE
Feature/us9094 filter search on day time frequency v3

### PR DESCRIPTION
**Pull request by vchtelmakh**

### CODE REVIEW

File: src/app/shared/constants.ts [hunk 1]
```diff
@@ -121,16 +181,31 @@ export const maxZoom: number = 15;          // The maximum zoom before zoomAdjus
...
-            meetingFrequencyId: 1,
-            meetingFrequencyDesc: 'Every week'
-        }, {
-            meetingFrequencyId: 2,
-            meetingFrequencyDesc: 'Every other week'
-        }, {
-            meetingFrequencyId: 8,
-            meetingFrequencyDesc: 'Every month'
-        }];
+  meetingFrequencyId: 1,
```
Comment (JoeKer1) [inline src/app/shared/constants.ts]: I get it, but this just feels weird.

File: src/app/models/simple-selectable.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export class SimpleSelectable {
```
Comment (JoeKer1) [inline src/app/models/simple-selectable.ts]: Can/Did you replace existing models with this one?

Comment (vchtelmakh) [inline src/app/models/simple-selectable.ts]: I didn't, though I probably could. I think it should be done in a refactor story though. I am making a refactor story to remove redundancy in the filter component, it's silly.

File: src/app/shared/constants.ts (hunk 1 above)
Comment (vchtelmakh) [inline src/app/shared/constants.ts]: If you mean the names being passed in - I am using them elsewhere, so I want to eliminate hard-coding in more than one place.

**Pull request merged**